### DISCUSSION TITLE
Good news! Package on Pypi!

### DISCUSSION THREAD

**Discussion by rdbende** [rdbende/Sun-Valley-ttk-theme#19] [Announcements]
I created a simple package to make this theme easy to use.

### Installation
```
pip install sv-ttk
```

### Documentation
```python
import tkinter as tk
import sv_ttk

root = tk.Tk()

sv_ttk.set_theme("light")  # Set light theme
sv_ttk.set_theme("dark")  # Set dark theme

sv_ttk.use_light_theme()  # Set light theme
sv_ttk.use_dark_theme()  # Set dark theme

sv_ttk.toggle_theme()  # Toggle dark and light theme

print(sv_ttk.get_theme())  # Get what theme the app uses

root.mainloop()
```

**Comment by HuyHung1408** [comment]
@rdbende How can I change the font size with this one? The font seems to be too small for me

**Reply by rdbende** [reply]
You can use Tkinter's standard `option_add` method

```python
root.option_add("*font", ("Segoe UI", 12))  # Change default font and size

root.option_add("*font", "-size 20")  # Just change the size of the currently used font
```

**Reply by rdbende** [reply]
The theme uses Segoe UI 10 px by default

**Reply by HuyHung1408** [reply]
Thanks a lot!

**Comment by xiaoxlh** [comment]
How do I import this theme?

**Reply by rdbende** [reply]
Scroll up ca. 800 pixels, and there it is:

> import sv_ttk

**Comment by fbarre96** [comment]
How do I disable / change completely the style for one element?
This code is not working so far:

```python
import tkinter as tk
from tkinter import ttk

import sv_ttk

root = tk.Tk()
style = ttk.Style()
sv_ttk.use_dark_theme()
style.configure("Other.TButton", background="red", borderwidth=1, bordercolor="green")
btn = ttk.Button(text="John")
other_btn = ttk.Button(text="Doe")
other_btn.configure(style="Other.TButton")
btn.pack()
other_btn.pack()

root.mainloop()
```

**Reply by rdbende** [reply]
Oh, I didn't see this. Next time please open a separate discussion, I'll notice it sooner. 😏

Unfortunately it's not possible to disable a single style element, if you really need that, you can use plain Tkinter widgets.

Since this is a pixmap-theme, changing colors of a style isn't as straightforward, as just throwing `background` and `bordercolor` arguments to `style.configure`, instead you have to modify the images in `theme/light/` and `theme/dark` (#12), which is kinda bleh...

**Reply by fbarre96** [reply]
I ended up with a not so good solution but it's working. I define I Default.TButton element with a custom layout without the image component in it:

```python
style = ttk.Style(tkApp)
sv_ttk.set_theme("light")  # Set light theme

# Set a layout for buttons called Default.TButton
style.layout("Default.TButton", [('Button.border', {'sticky': 'nswe', 'border': '1', 'children': [('Button.focus', {'sticky': 'nswe', 'children': [('Button.padding', {'sticky': 'nswe', 'children': [('Button.label', {'sticky': 'nswe'})]})]})]})])
# Style your button
style.configure("Default.TButton", relief="sunken",background="#73B723",
                     foreground="white", font=('Sans', '10', 'bold'), borderwidth=0)
# Use it
btn = ttk.Button(tkApp, text="button",  style="Default.TButton")
# or use theme buttons
btn = ttk.Button(tkApp, text="stylish button")
```
The layout used here was extracted from the theme "clam".

**Reply by rdbende** [reply]
Yeah, this seems like it's the best solution actually.
